integration_tests/steps/power_tests.py

```diff
@@ -34,12 +34,20 @@ def test_baseline_peak_energy_stats(context):
         context.simulation.endpoint_buffer.area_throughput_stats.results
 
     expected_results = {'Neighborhood 1': {'import': {'peak_energy_kWh': 0.4,
-                                                      'peak_percentage': 100.0}},
+                                                      'peak_percentage': 100.0,
+                                                      'capacity_kwh': 2.0,
```

Comment:
It is better to stick to the common naming and uppercase the 'W' on 'kWh'. Therefore 'kwh' -> 'kWh'